app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusBundleMoreView.kt

```diff
@@ -0,0 +1,340 @@
+  uiState: AppsListUiState,
+  title: String,
+  bundleTag: String,
+  reload: () -> Unit,
+  noNetworkReload: () -> Unit,
+  navigateBack: () -> Unit,
+  navigate: (String) -> Unit,
+) {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth()
+      .fillMaxHeight(),
+    horizontalAlignment = Alignment.CenterHorizontally
+  ) {
+    AppGamesTopBar(navigateBack = navigateBack, title = title)
+    when (uiState) {
+      AppsListUiState.Loading -> LoadingView()
+      AppsListUiState.NoConnection -> NoConnectionView(onRetryClick = noNetworkReload)
+      AppsListUiState.Error -> GenericErrorView(reload)
+      AppsListUiState.Empty -> EmptyView(text = "")
+
+      is AppsListUiState.Idle -> {
+        val appList = uiState.apps.onEach {
+          it.campaigns?.run {
+            if (AptoideMMPCampaign.allowedBundleTags.keys.contains(bundleTag)) {
+              placementType = "see_all"
+            }
+          }
+        }
+        LazyColumn(
+          modifier = Modifier
+            .semantics { collectionInfo = CollectionInfo(appList.size, 1) }
+            .wrapContentSize(Alignment.TopCenter)
+        ) {
+          item { MoreBonusSectionView(title = title) }
+          itemsIndexed(appList) { index, app ->
+            AppItem(
+              modifier = Modifier.padding(horizontal = 16.dp),
+              app = app,
+              onClick = {
+                navigate(
+                  buildAppViewRoute(app.packageName)
+                    .withItemPosition(index)
+                )
+              },
+            ) {
+              InstallViewShort(app)
+            }
+          }
+          item { WantMoreSectionView() }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun MoreBonusSectionView(title: String) {
+  Box(
+    modifier = Modifier
+      .padding(vertical = 24.dp)
+  ) {
+    val percentage = title.split(" ")[2].dropLast(1)
+    val splitText = stringResource(id = R.string.bonus_page_body_1, "%s", percentage)
+    val annotatedString = buildAnnotatedString {
+      append(splitText)
+    }
+    val inlineContent = mapOf(
+      "%s" to InlineTextContent(
+        placeholder = Placeholder(
+          width = 16.sp,
+          height = 16.sp,
+          placeholderVerticalAlign = PlaceholderVerticalAlign.TextBottom
+        ),
+        children = {}
+      )
+    )
+    Image(
+      imageVector = getBonusOnEveryPurchase(
+        iconColor = Palette.Primary,
+        outlineColor = Palette.Black,
+        backgroundColor = Palette.Secondary,
+        themeColor = Palette.Black
+      ),
+      contentDescription = "Bonus Section",
+      modifier = Modifier
+        .fillMaxWidth(),
+      contentScale = ContentScale.FillWidth,
+    )
+    AptoideOutlinedText(
+      text = title,
+      style = AGTypography.InputsM,
+      outlineWidth = 14f,
+      outlineColor = Palette.Black,
+      textColor = Palette.Primary,
+      modifier = Modifier
+        .padding(start = 56.dp, top = 14.dp)
+    )
+    Column {
```

Comment:
The horizontal padding could be applied to the column instead of each of the children